### PR TITLE
fix(dialog): Improve accessibility

### PR DESCRIPTION
**Pull request by acdvorak**
- Allows keyboard users to scroll the content of the `--scrollable` dialog:
    - Adds a focusable element to the dialog's content
    - Adds a 1px outline on focused links within dialog content
    - Changes dialog content from Latin ("Lorem Ipsum") to English ("The Wonderful Wizard of Oz")
- Makes "OK" button the default action in Confirmation dialog:
     - Pressing the <kbd>Enter</kbd> key when a list item has focus will now close the dialog

image

### CODE REVIEW

File: src/DialogCatalog.js [hunk 1]
```diff
@@ -418,18 +418,16 @@ class ScrollableDialog extends Component {
...
-                <p>Molestiae sapiente quae nulla. Voluptates quibusdam numquam earum vero deserunt in, cum tenetur accusamus ipsum minus veniam libero quasi fuga dolorem laudantium error quo et accusantium neque vitae aliquam tempore.</p>
-                <p>Optio asperiores quisquam odit eaque incidunt laboriosam repudiandae ex eum iure quia, id vero atque perspiciatis, officiis quaerat aut ut dolorem libero eos perferendis ducimus! Veritatis nam libero tempora maxime?</p>
-                <p>Sapiente reiciendis quis eveniet iure dicta perferendis quos consectetur, soluta sunt, labore ipsam inventore maiores laudantium recusandae deleniti autem animi consequatur, voluptatem sint. Dignissimos minima labore earum vitae ad non!</p>
-                <p>Cum ex totam dolore officiis maiores quidem necessitatibus consequatur molestias culpa, quas, aperiam tempora et! Dolorem, voluptates dignissimos? Voluptatem voluptatibus expedita, error ducimus distinctio necessitatibus laudantium officiis dolorum nam vitae?</p>
+                <p><a href='http://www.gutenberg.org/ebooks/55'>Read the full book</a></p>
```
Comment (kfranqueiro) [inline src/DialogCatalog.js]: Add `target='_blank'`

File: src/styles/DialogCatalog.scss [hunk 2]
```diff
@@ -4,3 +4,7 @@
   position: relative;
   z-index: 0;
 }
+
+.mdc-dialog__content a:focus {
+  outline: 1px solid currentColor;
```
Comment (kfranqueiro) [inline src/styles/DialogCatalog.scss]: I'm confused why this is needed... shouldn't focused links be outlined by default?

File: src/DialogCatalog.js (hunk 1 above)
Comment (acdvorak) [reply src/DialogCatalog.js]: Done

File: src/styles/DialogCatalog.scss (hunk 2 above)
Comment (acdvorak) [reply src/styles/DialogCatalog.scss]: They definitely _should_ be outlined by default, but for some reason Chrome doesn't render it on this page, whereas Firefox and Safari _do_. I don't want a silly Chrome bug to hold up our accessibility review.

**Pull request merged**